build.sh

```diff
@@ -148,4 +148,12 @@ echo Generating $BUILD_TYPE makefiles
 cmake $CMAKE_GEN $CC_PREFIX $ICU_PATH -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
 
 $MAKE $MULTICORE_BUILD 2>&1 | tee build.log
+
+result=$?
+if [[ $result == 0 && ! -f ../ch ]]; then
+    ln -s BuildLinux/ch ../ch
```

Comment:
>  I tried it out and this broke my personal settings. I have "." at head of PATH. When I build at repo root and run "ch", it prints

local one is available at `./ch`

> I agree with the above concerns. It's fairly straightforward to know where the test binary is.

you can call `ls -l ch`

> We might consider build type subdirs like BuildLinux/x64_debug, 

We could have `ch_d` or `ch_debug` etc..

> Especially as long as this is intended to use with already-written test code, saving typing is a moot point.

Sorry, didnt' understand. Could you please explain more?